### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Itswali**
### In This PR.

- Install renderer and reacat testing library
- create snap tests for all components functions
- create unit test for operate and calculate files

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
 export default App;
+
+{ /* <nav className="navbar">
+      <div className="Title-name">MATH MAGICIANS</div>
+      <div className="nav-links">
+        <a href="/Home">HOME</a>
+        <a href="/Calculator">Calculator</a>
+        <a href="/Quote">QUOTE</a>
+      </div>
+    </nav> */ }
```
Comment (codecaiine) [inline src/App.js]: - [x] I noticed that you comment code here.  :cry:  Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Itswali @tsohleDev , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Great testing setup :heavy_check_mark: 
- No linters errors :heavy_check_mark: 


 ## Required Changes ♻️
- [ ] Please make sure that when you remove the commented code it is not affect your tests :+1: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Itswali and @tsohleDev,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tsohleDev** [comment]
Thank you @codecaiine and @DammyShittu

**Pull request merged**